ffi/derive/tests/ui_fail/nested_owned_structure.rs

```diff
@@ -1,21 +1,18 @@
 use std::mem::MaybeUninit;
 
-use iroha_ffi::{ffi_export, slice::OutBoxedSlice, FfiType};
-
-type Nested = Vec<Vec<FfiStruct>>;
+use iroha_ffi::{ffi_export, FfiType};
 
 /// Ffi structure
 #[derive(Clone, FfiType)]
 pub struct FfiStruct;
 
 /// Return nested structure
 #[ffi_export]
-pub fn return_nested() -> Nested {
+pub fn return_nested() -> Vec<Vec<FfiStruct>> {
```

Comment:
Consider turning this into a doctest, and hard-coding the error code.

Reply:
I'd rather leave it for if/when we'll be moving all ui tests to doc tests